src/account/interface.ts

```diff
@@ -45,11 +45,16 @@ export abstract class AccountInterface extends ProviderInterface {
    * - entrypoint - the entrypoint of the contract
    * - calldata - (defaults to []) the calldata
    *
+   * @param estimateFeeDetails -
+   * - optional blockIdentifier
+   * - constant nonce = 0
+   * @param addsAbstraction an array of BigNumberish, used as additional parameters for account abstraction, for message hash and signature.
    * @returns response from estimate_fee
    */
   public abstract estimateInvokeFee(
     calls: AllowArray<Call>,
-    estimateFeeDetails?: EstimateFeeDetails
+    estimateFeeDetails?: EstimateFeeDetails,
```

Comment:
could we push everything related to `...addsAbstraction` inside the `estimateFeeDetails` ? The same applies for all other methods here
cc: @tabaktoni

Reply:
The number of parameters of abstraction are unknown.
Should be replaced by an array.
My feeling is that it will complicate the user experience.